tests/kvmtest.sh

```diff
@@ -142,6 +142,7 @@ test_t0() {
       ssh/test_ssh_ciphers.py \
       ssh/test_ssh_limit.py \
       syslog/test_syslog.py\
+      syslog/test_syslog_counter.py\
```

Comment:
This kvmtest.sh file is no longer being used in PR testing. No need to update it.